tests/tests_sign_workflow/test_sign_artifacts.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from pathlib import Path
+
+from sign_workflow.sign_artifacts import SignArtifacts, SignArtifactsExistingArtifactFile, SignExistingArtifactsDir, SignWithManifest
+
+
+class TestSignArtifacts(unittest.TestCase):
+
+    def test_signer_class_method(self):
+        path = Path(r"/dummy/path/manifest.yml")
+        klass = SignArtifacts.__signer_class__(path)
+
+        self.assertEqual(type(SignWithManifest), type(klass))
+
+        path = Path(r"/dummy/path/")
```

Comment:
Split up in separate tests.

Reply:
I think it should be one method, since it just verifies multiple if conditions with no workflow. It would just increase verbosity and complexity. This is clear too. Thoughts?

Reply:
It's not a must have, but my reasoning is that these tests can fail independently, and methods are cheap.